partOne/src/app/app.component.ts

```diff
@@ -49,11 +49,13 @@ export class AppComponent implements OnInit {
 
     // Checks if should display install popup notification:
     if (isIos() && !isInStandaloneMode()) {
-      this.toast.openFromComponent(IosInstallComponent, { 
-        duration: 8000,
-        horizontalPosition: 'start', 
-        panelClass: ['mat-elevation-z3'] 
-      });
+      setTimeout(() => {
+          this.toast.openFromComponent(IosInstallComponent, {
+            duration: 8000,
+            horizontalPosition: 'start',
+            panelClass: ['mat-elevation-z3']
+          });
+      }, 100);
```

Comment:
The 100 is actually not needed. It just requires the next tick of render which settimeout will provide on its own. Can you remove the 100 and I'll get this merged? Thanks!

Reply:
Cool. Removed. :)